### DISCUSSION TITLE
Why Images are not loading

### DISCUSSION THREAD

**Discussion by Vishal-beep136** [community/community#40678] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

## Issue -

Some images aren't loading despite the fact I do know that the image exists.

images aren't loading in the readme and the repo directory (assets/pics ).

## Proof

for example in this readme screenshots are not loading [sangcomz/FishBun](https://github.com/sangcomz/FishBun)
and in this pic directory, none of the pics is loading [FishBun's pic](https://github.com/sangcomz/FishBun/tree/master/pic)

here are screenshots  - 

For Readme - 

![Screenshot 2022-12-03 133603](https://user-images.githubusercontent.com/82146140/205431284-d4f260d3-ed78-4adb-87a3-ac0969a55b28.png)

For the repo's directory (pics) -

![Screenshot 2022-12-03 133951](https://user-images.githubusercontent.com/82146140/205431413-a762e759-894e-402b-b254-e8f0fefba483.png)

The same happens for all images including `light1.png, dark3.png`, and for different repos, images as well.

I've also tried switching browsers to edge, chrome, brave didn't work either.

I guess It is only happening on my device

# Edit 
I found that if I use VPN then this issues is fixed! but don't want that!

**Comment by jfullstackdev** [comment]
yup, I'm seeing same issues here, that's either changing network or using VPN 

and as far as I know, GitHub is not the one selecting this mode, rather it's the ISP

**Reply by Vishal-beep136** [reply]
But I've used VPN browser extension in different browser, not in main browser does that also count ?

**Comment by krshrimali** [comment]
I'm also facing this issue - while uploading the images. Example: https://github.com/krshrimali/blog/blob/daily_reports/assets/blogs/Autograd-Example-2.png (doesn't load on my system). :/

**Reply by Vishal-beep136** [reply]
This is kind of frustrating I've to use VPN to load images :(
